AEPMessaging/Sources/Message+FullscreenMessageDelegate.swift

```diff
@@ -33,6 +34,7 @@ extension Message: FullscreenMessageDelegate {
         guard let message = message.parent else {
             return
         }
+        message.recordEventHistory(eventType: .dismiss, interaction: nil)
```

Comment:
Should we record the IAM history even if showing it failed? If the frequency rule is, say show once, the IAM would never get shown in this case